skbio/metadata/_intersection.pyx

```diff
@@ -0,0 +1,513 @@
+"""
```

Comment:
@jairideout , I verified the change is not necessary. However, the change is only an addition of a `update()` function, which is unit tested. Is it ok to leave it there in case it is useful for future? Plus, I mentioned the change in the head of those imported files.